.gitignore

```diff
@@ -22,6 +22,6 @@ result-*
 output.js
 .DS_Store
 .idea/
-test-data/chrome-user-data/
+test-data
```

Comment:
Not like that. We should add a `.gitkeep` in test-data to make sure the user does not have to create the directory manually.

Reply:
Same goes for the template.